sphinx/source/docs/installation.rst

```diff
@@ -30,12 +30,20 @@ dependency on several compiled libraries.
 Ideally, you should have the following libraries installed:
 
 * NumPy
+* Pandas
 * Flask
 * Redis
+* Redis-py
+* Six
 * Requests
-* gevent
-* gevent-websocket
-* Pandas
+* Tornado >= 4.0
+* Werkzeug
+* Greenlet
+* Gevent
+* Gevent-websocket
+* PyZMQ
+* PyYaml
+* DateUtil
 
```

Comment:
Oh... you did it... thanks! It would be nice to add somewhere something about the testing dependencies...